### DISCUSSION TITLE
Empreinte carbone du tabagisme

### DISCUSSION THREAD

**Discussion by OliC-BE** [incubateur-ademe/nosgestesclimat#1553] [Ideas] [answered]
Je pensais ajouter qq question pour les fumeurs puisque
https://www.who.int/fr/news/item/31-05-2022-who-raises-alarm-on-tobacco-industry-environmental-impact

https://www.greenly.earth/blog-fr/quelle-est-l-empreinte-carbone-de-l-industrie-du-tabac#:~:text=Une%20cigarette%20c%27est%20%3A,et%20de%20l%27oxyde%20nitreux.

Ce qui suggère que c'est aussi impactant que la consommation de viande.

Merci

**Comment by laem** [comment, marked as answer]
Je ferme au profit de https://github.com/datagir/nosgestesclimat/issues/135
